_drafts/oryx/technology.md

```diff
@@ -0,0 +1,31 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+It is built on a variety of technologies and in this article, we will take a closer look at each of these technologies and see how they contribute to the development of Oryx.
+
+## HTML
+HTML is the backbone of the web. It is the markup language used to create web pages and applications. In Oryx, HTML is used to structure the content and layout of the application. It provides a semantic structure to the page, making it easier to understand and maintain.
+
+## CSS
```

Comment:
For CSS we could add here:
- we use standard CSS, no BEM, no preprocessors, no postCSS. (we'll add more on CSS in a separate section,we can link it together later)